railgun/manager/run.go

```diff
@@ -57,6 +59,7 @@ func Run(ctx context.Context, c *config.Config) error {
 	utilruntime.Must(konghqcomv1.AddToScheme(scheme))
 	utilruntime.Must(configurationv1alpha1.AddToScheme(scheme))
 	utilruntime.Must(configurationv1beta1.AddToScheme(scheme))
+	utilruntime.Must(knativev1alpha1.AddToScheme(scheme))
```

Comment:
Above I was talking about adding knative to the scheme only in the runtime and not during package load, this line would seem to do just that so I expect the other files can be dropped.

Reply:
Agree. should be able to cleanup.